src/store/buoy.ts

```diff
@@ -139,6 +141,67 @@ const createStore = (config: BuoyConfig, bustCache = false) => {
         }
       },
     },
+    getters: {
+      dates(): Date[] {
+        // unique sorted dates
+        return this.samples
+          .map(({ date }) => date)
+          .filter((v, i, a) => a.indexOf(v) === i)
+          .sort((f, s) => f.valueOf() - s.valueOf());
+      },
+      minDate(): string {
+        return localISODate(this.minDateRaw);
+      },
+      maxDate(): string {
+        return localISODate(this.maxDateRaw);
+      },
+      siteCoordinates() {
+        return function (site: string) {
+          const match = this.coordinates.find(
+            ({ station_name }) => station_name === site
+          );
+          if (match) {
+            return [match.longitude, match.latitude];
+          } else {
+            return [0, 0];
+          }
+        };
+      },
+      selectedSamples() {
+        const daySamples = this.samples.filter(
+          ({ date }) => date - this.maxDate === 0
+        );
```

Comment:
In what format is date and this.maxDate here? 

If they are both strings, then you could do `"2022-01-01" === "2022-01-01"` here instead.

`date === this.maxDate`.

If not, you may need to convert the dates to something you can compare.

For example `Date.now()` gives me a number, BUT is more granular than a day.

Reply:
this part confuses me a bit, maybe we can chat about it next time we are on a call?